src/components/speakerDetails.re

```diff
@@ -0,0 +1,25 @@
+open Util;
+
+open Data;
+
+[@bs.module] external style : Js.t({..}) = "./speakerDetails.module.scss";
+
+let component = ReasonReact.statelessComponent("SpeakerDetails");
+
+let make = (~speaker: Data.speakerData, _children) => {
+  ...component,
+  render: _self =>
+    <div className=style##root key=speaker.name>
```

Comment:
`key` isn't needed here since you'll need to add it in the loop

Reply:
good eye!